src/components/main-page/About/About.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from "react";
+import styles from "./About.module.scss";
+import { Section, Heading, Text, Card } from "components/ui";
+import useTranslation from "translations/hooks/useTranslations";
+
+export function About() {
+  const { t } = useTranslation();
+
+  const paragraphs = t("about.paragraphs") as string[];
+
+  return (
+    <Section variant="white" id="about">
+      <Heading withAccent>{t("about.title")}</Heading>
+      <div className={styles.textContainer}>
+        {paragraphs?.length > 0 &&
+          paragraphs.map((p: string, index: number) => (
+            <Text className={styles.textField} key={index}>
+              {p}
+            </Text>
+          ))}
```

Comment:
I'm slightly hesitant agains using arrays as values of translation files - haven't heard of such a feature in any of the libraries and I feel like using key/value pairs with values being `string` and some meaningful `key` names could be more readable. You mentioned, that you've seen the use of arrays of translations somewhere - I'd be more than happy if you'd add some link to example in which it's used

Reply:
I will wait with this until you push your changes from #44 😉